frontend/src/components/SignUp.js

```diff
@@ -179,7 +182,20 @@ export class SignUp extends React.Component {
                             name='error_uri'
                             value={this.props.location.pathname}
                         />
-                        <button type='submit'>Sign Up &raquo;</button>
+                        <div style={{ display: this.state.captchaPassed && 'none' }}>
```

Comment:
I think we also have to configure a reCAPTCHA secret key and server-side validation to do the reCAPTCHA validation properly -- see https://developers.google.com/recaptcha/docs/verify

Reply:
Yes, I have added the logic in the latest commit. Thanks for pointing it out.